### ISSUE TITLE
Consider adding on_post_command{} section

### ISSUE THREAD

**Issue by warrigal24** [ainslec/adventuron-issue-tracker#246]
We currently have an `on_pre_command{}` and `on_command{}` section, but no `on_post_command{}` section.

My vision is that `on_pre_command{}` is used for pre-processing of commands, i.e. things like checking that a transitive verb has a noun and an intransitive verb doesn't, coercing certain commands into another form (e.g. GET ON may be coerced into MOUNT or ENTER), possibly doing early disambiguation and checking that the object is present or carried or beside or whatever.

The `on_command{}` section is used to handle specific cases. If the pre-processing has been handled correctly, this means that the number of checks in this section is minimised, as you don't have to duplicate code for things that have already been checked during pre-processing.

The `on_post_command{}` section is used for post-processing to handle the default cases that weren't handled by the specific cases in `on_command{}`.

Verbs that don't have any specific cases (like ABOUT) can be handled in the pre-processing or the post-processing, depending on the author's preference.

The whole idea behind this is code organisation. At the moment, `on_command{}` can be a pack of poo ticket. The suggested structure allows for cleaner organisation of code.

**Comment by ainslec** [comment]
Added as **experimental_on_command_2 {}** in beta 70.

The naming convention of on_post_* is already taken with on_post_tick, but the danger of naming this as on_post_command, is that it perhaps will lead to users believing it has the same behaviour.

on_post_tick is a completely separate block that ALWAYS runs after executing on_tick (except where a redescribe is issued from on_tick, and another tick follows the redescribe tick).

"on_post_command" would be different as it really represents default handlers, and they only execute if something hasn't been executed earlier (such as in on_command {}).

on_command_2 {} is probably the most reasonable name I can think of, as it implies its a continuation of on_command, rather than an event that happens after on_command {}.

As I'm not certain that the name will be liked, or that there is not a better name available, for now, I have named it experimental_on_command_2 {}, which of course is an awful name. But it allows the feature to be tested, until a good name is decided upon.
